documentation/Functions.md

```diff
@@ -185,9 +185,11 @@ createDevon4jProject("com.mycustomer.myapplication")
 ***
 
 ### buildJava <a name="buildJava"></a>
+This function will build your java project.
 #### parameter 
-1. The project directory, relative to workspace.
-2. (Optional) Indicator whether tests should be run. Default is false.
+There are two parameter you need to use. 
```

Comment:
You need to use just one parameter the other is optional, maybe say the function consist of 2 parameters and after that the parameter definition as a list.